draft-ietf-rats-corim.md

```diff
@@ -166,11 +166,19 @@ This specification describes the CoRIM format and documents how a Verifier shoul
 A Verifier needs to reconcile its various inputs, with CoRIM being one of them.
 In addition to the external CoRIM documents, the Verifier is expected to create an internal representation for each input and map each external representation to an internal one.
 By using the internal representation, the Verifier processes inputs as if they are part of a conversation, keeping track of who said what.
-The origin of the inputs is tracked as *authority*.
-The authority for the Claims in a CoRIM is the CoRIM issuer.
 
 Effectively, Attesters, Reference Value Providers, Endorsers, Verifier Owners, Relying Parties, and even the Verifier potentially all contribute to the conversation.
-Each producer of corresponding RATS Conceptual Messages can assert Claims about an Attester's actual or allowed state.
+These individual principals of the conversation may have their identity rooted back to some *authority* farther up the certificate chain than the signing key itself, such as a digital certificate authority intermediate or root key.
+The origin of contributions/inputs to the conversation are tracked at the granularity of authority, so CoRIM authors can predicate their reference measurements or endorsements based on existing properties rooted from the same authority.
+
+CoRIM inputs to the conversation include measurements within a specific Environment.
+A measurement never appears outside the scope of an associated Environment, so the Environment-Claim Tuple (ECT) is a data structure used to encapsulate all scope context for a given set of measurements.
+The authority for an ECT from a CoRIM is the CoRIM issuer.
+The collective content of an ECT may be referred to as a property, claim, or assertion when the content need not be presented in the ECT data structure.
+
+Important: The Verifier's objective is to produce a list of properties that describe the Attester's presumed actual state.
+
+Each producer of corresponding RATS Conceptual Messages can assert claims about an Attester's actual or allowed state.
```

Comment:
Are you also implying that Evidence can assert claims about allowed state?

Reply:
Conceptual messages aren't all evidence. I thought there was already a cm-ind allocated for policy?